### PR TITLE
New Nathaniel Cho cards released by Whisperer in Darkness

### PR DESCRIPTION
**Pull request by robert-huang**

### CODE REVIEW

**Comment by robert-huang** [comment]
source: https://www.youtube.com/watch?v=flUQtWv-pkY

**Comment by drawntotheflame** [comment]
hey @robert-huang I'm going to add these onto the DttF announced cards and the forthcoming MB announced cards this weekend. Do you want to be involved in the scribes of arkhamdb? Are you on the MB discord?

**Comment by robert-huang** [comment]
Sure, I was changing something else and thought I'd do this one too, but I'll join in - I'm Robert#5742 on discord.

**Comment by robert-huang** [comment]
Sure, I was changing something else and thought I'd do this one too, but I'm definitely down to join in! - I'm Robert#5742 on discord.

**Comment by drawntotheflame** [comment]
I've added you there @robert-huang :)

File: pack/investigator/nat.json [hunk 1]
```diff
@@ -33,18 +33,21 @@
 		"deck_limit": 1,
 		"faction_code": "guardian",
 		"health": 1,
-		"illustrator": "Patrick McEvoy",
+		"flavor": "You gotta give this up, Nathaniel. Sooner or later, you're gonna get yourself killed.",
```
Comment (drawntotheflame) [inline pack/investigator/nat.json:36]: Please add '\"' (double speechmarks) at the start and end of the quotation.

File: pack/investigator/nat.json [hunk 2]
```diff
@@ -33,18 +33,21 @@
...
 		"restrictions": "investigator:60101",
-		"sanity": 1,
+		"sanity": 3,
+		"skill_intellect": 1,
 		"skill_wild": 1,
+		"skill_willpower": 1,
 		"slot": "Ally",
-		"text": "Nathaniel Cho deck only.\n<b>This card has not yet been announced and is a placeholder.</b>",
-		"traits": "Ally.",
+		"text": "Nathaniel Cho deck only.\n[reaction] After Randall Cho enters play: Heal 3 damage or search your deck and discard pile for a [[Weapon]] asset, play it (paying its cost), and shuffle your deck.",
```
Comment (drawntotheflame) [inline pack/investigator/nat.json:49]: Please change 'reaction' to 'fast'.

Comment (drawntotheflame) [reply pack/investigator/nat.json:49]: Please add italics tags around the parenthetical text.

File: pack/investigator/nat.json [hunk 3]
```diff
@@ -33,18 +33,21 @@
...
 		"pack_code": "nat",
 		"position": 2,
 		"quantity": 1,
 		"restrictions": "investigator:60101",
-		"sanity": 1,
+		"sanity": 3,
+		"skill_intellect": 1,
 		"skill_wild": 1,
+		"skill_willpower": 1,
 		"slot": "Ally",
```
Comment (drawntotheflame) [inline pack/investigator/nat.json:48]: please add 'subname: "Concerned Brother",'

**Comment by robert-huang** [comment]
> I've added you there @robert-huang :)

I don't think I got anything, did you add me to a channel or as a friend?

**Comment by drawntotheflame** [comment]
> > I've added you there @robert-huang :)
> 
> I don't think I got anything, did you add me to a channel or as a friend?

I added you as a friend (I'm zooeyglass on there) so that I can add you to our scribes group.

**Comment by robert-huang** [comment]
Hmm, I can't see it in my pending, but I've sent you a friend request so hopefully that works!

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live. Thanks

**Pull request merged**